variables.robot

```diff
@@ -233,6 +233,10 @@ ${OS_UBUNTU}=               ubuntu
 &{RTE65}=                   ip=192.168.10.54
 ...                         platform=optiplex-7010
 ...                         platform_vendor=DELL    sonoff_ip=192.168.10.250
+# 3mdeb Protectli VP2430 -----------------------------------------------------
+&{RTE68}=                   ip=192.168.10.18
```

Comment:
Why cannot it be in ascending order? Now RTE68 is between RTE65 and RTE66...

Reply:
updated
https://github.com/Dasharo/open-source-firmware-validation/pull/554/commits/339aabef454c83c257753f00b2b75a3cf28ffa3b
I forgot to rearrange the items after merging.